python/example_code/bedrock-agent-runtime/bedrock_agent_runtime_wrapper.py

```diff
@@ -68,4 +68,60 @@ def invoke_agent(self, agent_id, agent_alias_id, session_id, prompt):
 
     # snippet-end:[python.example_code.bedrock-agent-runtime.InvokeAgent]
 
-# snippet-end:[python.example_code.bedrock-agent-runtime.BedrockAgentsRuntimeWrapper.class]
+        # snippet-start:[python.example_code.bedrock-agent-runtime.InvokeFlow]
+    def invoke_flow(self, flow_id, flow_alias_id, input_data, execution_id):
```

Comment:
Can we include type hints for these parameters?

Reply:
Added params as per the InvokeAgent example. Let me know if there's more needed.

Reply:
Also updated boto version.